web-ui/src/main/resources/catalog/components/userfeedback/partials/userfeedback.html

```diff
@@ -0,0 +1,37 @@
+<div>
```

Comment:
Indentation of html files looks incorrect, should be used 2 characters. Please make sure to use an IDE that uses `.editorconfig`

Reply:
Fixed